qpc/messages.py

```diff
@@ -308,6 +308,7 @@
 INSIGHTS_TMP_ERROR = 'An error occurred while saving a temporary copy of' \
     ' the QPC report.  Cannot write file to %s.'
 INSIGHTS_REQUIRE_SUDO = 'Insights upload command requires sudo access.'
+INSIGHTS_NO_GPG_HELP = 'Upload to insights without GNU Privacy Guard.'
```

Comment:
We should refer to Insights with a capital letter in this message (e.g. `Insights`)